resources/packs/okta/sdk/notifications.go

```diff
@@ -0,0 +1,62 @@
+package sdk
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+)
+
+type SecurityNotificationEmails struct {
+	// New sign-on notification email
+	SendEmailForNewDeviceEnabled bool `json:"sendEmailForNewDeviceEnabled"`
+	// Password changed notification email
+	SendEmailForPasswordChangedEnabled bool `json:"sendEmailForPasswordChangedEnabled"`
+	// MFA enrolled notification email
+	SendEmailForFactorEnrollmentEnabled bool `json:"sendEmailForFactorEnrollmentEnabled"`
+	// MFA reset notification email
+	SendEmailForFactorResetEnabled bool `json:"sendEmailForFactorResetEnabled"`
+	// Report suspicious activity via email
+	ReportSuspiciousActivityEnabled bool `json:"reportSuspiciousActivityEnabled"`
+}
+
+// GetSecurityNotificationEmails retrieves the security configuration
+func (m *SkdExtension) GetSecurityNotificationEmails(ctx context.Context, orgId string, token string, client *http.Client) (*SecurityNotificationEmails, error) {
+	// This code was derived from https://github.com/okta/terraform-provider-okta/blob/master/sdk/security_notification_emails.go
+
+	// we need to split the orgId into orgName and domain because this API uses a different domain
+	orgName, domain, found := strings.Cut(orgId, ".")
+	if !found {
+		return nil, errors.New("cound not determine orgName and domain from orgId " + orgId)
+	}
+	url := fmt.Sprintf("https://%s-admin.%s/api/internal/org/settings/security-notification-settings", orgName, domain)
```

Comment:
It is uses by Okta in their official terraform provider. I agree that we should minimize the calls to internal apis as much as possible. In this case I consider it stable now since terraform uses it.